src/common/styles/Table.tsx

```diff
@@ -2,42 +2,32 @@ import styled from "@emotion/styled";
 import tw from "twin.macro";
 
```

Comment:
i think this entire file can safely delete already, good job

Reply:
done.